tests/test_module_loader.py

```diff
@@ -127,3 +128,128 @@ def test_v1_module_depends_on_v1_and_v2_module(tmpdir: py.path.local, snippetcom
     output = capsys.readouterr().out
     assert "Print from v2 module" in output
     assert "Hello world" in output
+
+
+@pytest.mark.parametrize("allow_v1", [True, False])
+def test_load_module_v1_already_installed(
+    project_builder: Callable[[str, List[str], List[str]], Project], modules_dir: str, allow_v1: bool
+) -> None:
+    """
+    Test whether the Project.load_module() method works correctly when loading a V1 module that was already installed
+    in the module path.
+    """
+    module_name = "elaboratev1module"
+    module_dir = os.path.join(modules_dir, module_name)
+    project: Project = project_builder(main_cf_content=f"import {module_name}", copy_to_libs_dir=[module_dir])
+
+    assert module_name not in project.modules
+    if allow_v1:
+        project.load_module(module_name=module_name, install=False, allow_v1=allow_v1)
+        assert module_name in project.modules
+    else:
+        with pytest.raises(ModuleNotFoundException, match=f"Could not find module {module_name}"):
+            project.load_module(module_name=module_name, install=False, allow_v1=allow_v1)
+
+
+def test_load_module_v1_module_using_install(project_builder: Callable[[str, List[str], List[str]], Project]):
+    """
+    Test whether the Project.load_module() method works correctly when a module is only available as a V1 module
+    and that module is not yet present in the module path.
+    """
+    module_name = "std"
+    project: Project = project_builder(main_cf_content=f"import {module_name}")
+    assert module_name not in project.modules
+    assert module_name not in os.listdir(project.downloadpath)
+    project.load_module(module_name=module_name, install=True, allow_v1=True)
```

Comment:
`install=True` is only relevant for v2 modules. For backwards compatibility, v1 modules are always allowed to be installed on the fly.

Reply:
I changed it to `False`.